src/App.js

```diff
@@ -1,11 +1,18 @@
-import Calculator from './components/calculator';
+import { Route, Routes } from 'react-router-dom';
+import HomePage from './pages/Home';
+import CalculatorPage from './pages/Calculator';
+import QuotePage from './pages/Quote';
+import Nav from './pages/Nav';
 
-function App() {
-  return (
-    <div className="App">
-      <Calculator />
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <Nav />
+    <Routes>
+      <Route exact path="/" element={<HomePage />} />
+      <Route path="/calculator" element={<CalculatorPage />} />
+      <Route path="/quote" element={<QuotePage />} />
+    </Routes>
+  </div>
+);
```

Comment:
- [ ] Everything is great, you followed SPA principles and app works as expected, only minor detail is remaining. Please make sure that the navbar is at the top of the page, currently it is sticked to body of the page and is dynamic:

Quotes | Calculator
------------- | -------------
<img width="1432" alt="Screenshot 2022-08-10 at 11 07 03" src="https://user-images.githubusercontent.com/77038610/183837153-6b1bbb5f-adf3-4c43-b695-2d1e08c7d8e3.png"> | <img width="1422" alt="Screenshot 2022-08-10 at 11 07 21" src="https://user-images.githubusercontent.com/77038610/183837214-28166598-998f-4024-8ae1-a6c1d73f4e58.png">



Wireframe: 

<img width="969" alt="Screenshot 2022-08-10 at 11 09 07" src="https://user-images.githubusercontent.com/77038610/183837503-9f081446-6eb6-4a3c-b425-4f7d4b216592.png">

Reply:
@ShoiraTa Thank you, I have removed the justify-content for the div from the CSS, and now the navigation is on top of the page.